components/collections-panel/index.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+import { addCollection, toggleFavourite, addResourceToCollection, removeResourceFromCollection } from 'redactions/user';
+import Component from './collections-panel-component';
+import { parseCollections, parseFavourites } from './collections-panel-selectors';
+
+const mapStateToProps = state => ({
+  collections: parseCollections(state),
+  favourites: parseFavourites(state)
+});
+
+const mapDispatchToProps = dispatch => ({
+  addCollection: collectionName => dispatch(addCollection(collectionName)),
```

Comment:
Don't do this... Let's pass the action as it is. If you change one parameter you will need to come to this file and change it too. This is a nest of bugs